torch/serialization.py

```diff
@@ -85,6 +87,14 @@
     MAP_SHARED, MAP_PRIVATE = None, None  # type: ignore[assignment]
 
 
+# _serialization_tls is used to store thread local state specific to serialization
+# that needs to be propagated to other files, in particular we use this for
+# (1) map_location (needed for wrapper subclasses/third party devices to torch._utils)
+# (2) skip_data (needed for torch.Tensor.__reduce_ex__ for skip_data ctx)
+# (3) materialize_fake_tensors (needed for torch.Tensor.__reduce_ex__ for skip_data ctx)
+_serialization_tls = threading.local()
```

Comment:
Ho I would have expected you just did `torch._C._stash_obj_in_tls("_serialization_tls", _serialization_tls)` here and nothing else below? 

Or if we don't know how to do it. It's ok to remote it all and open an issue as a TODO for later.

Reply:
Removed and opened issue here https://github.com/pytorch/pytorch/issues/134680